packages/victory-line/src/victory-line.tsx

```diff
@@ -150,5 +150,4 @@ export interface VictoryLineProps
   interpolation?: InterpolationPropType | Function;
   samples?: number;
   style?: VictoryStyleInterface;
-  animate?: boolean;
```

Comment:
Tested locally, works great!  Just a mistake from the TS migration, thanks for finding this.